src/sdk/api.ts

```diff
@@ -48,219 +46,164 @@ export class CyberusKeyAPI {
-   *
-   * @param {string} clientId Public client ID generated during creating the account.
-   * @param {string} redirectUri Redirect URI to which the response will be sent. If the value is not whitelisted then the request will fail.
-   * @param {OpenIdScopeParser} scope Each scope returns a set of user attributes, which are called claims.
-   *    Once the user authorizes the requested scopes, the claims are returned in an ID Token.
-   * @param {SoundEmitter} soundEmitter Interface which can play a sound.
-   * @param {Navigator} navigator Class describes an action that will be done to Authentication URL. For browsers it will be a page redirection.
-   * @param {string} [origin] The origin domain of the request being made. If `null` then the Referer header will be used.
-   * @param {string} [state]
-   *    RECOMMENDED. Opaque value used to maintain state between the request and the callback. Typically, CSRF, XSRF mitigation is done by cryptographically binding the value of this parameter with a browser cookie.
-   *    The state parameter preserves some state object set by the client in the Authentication request and makes it available to the client in the response.
-   *    It’s that unique and non-guessable value that allows you to prevent the attack by confirming if the value coming from the response matches the one you expect (the one you generated when initiating the request).
-   *    The state parameter is a string so you can encode any other information in it.
-   * @param {string} [nonce]
-   *    String value used to associate a Client session with an ID Token, and to mitigate replay attacks.
-   *    The value is passed through unmodified from the Authentication Request to the ID Token.
-   *    Sufficient entropy MUST be present in the nonce values used to prevent attackers from guessing values.
-   * @param {string} [responseType='code'] OpenId response type. The default is `code` (Code Flow, involving the front-channel and backchannel).
-   * @returns {Promise<void>}
-   * @memberof CyberusKeyAPI
-   */
-  public async authenticate(clientId: string, redirectUri: string, scope: OpenIdScopeParser, soundEmitter: SoundEmitter, navigator: Navigator, origin?: string, state?: string, nonce?: string, responseType = 'code'): Promise<void> {
-    if (this._geoProvider && !this._cachedGeo) {
-      this._cachedGeo = await this._geoProvider.getGeo();
+    /**
+     * Gets a URL with sound with embedded OTP. You have to emit it.
+     *
+     * @param {Session} session Cyberus Key's session generated by a user for a login.
+     * @returns {Promise<string>} string with url to the sound.
+     * @throws ResourceNotFoundError
+     * @memberof CyberusKeyAPI
+     */
+
+
+    public getOTPSound(session: string): Promise<string> {
+        const type = 'audio/mpeg'
+        const requestOptions = {
+            headers: {
+                'Accept': type,
+                'Content-Type': 'text/plain'
```

Comment:
text/plain ? Is it correct Content-Type for audio ? I think audio/mpeg could be better

Reply:
I followed previous implementation so I left what was there,  but according to the "book". 

1. This is a request parameter so content type is related to what is send, not to what is received (accept is related to what is received) 
2. By the book, content-type should be specified for post and put, and not get, so, it might be a better idea to delete it, but Artur wrote it that way and it was working, so i thought it was some hack related to backend. I'll check the consequences of deleting it, and delete it if that is really optional.

Reply:
Right, I didn't catch that is the GET request

Reply:
ok, so i'll leave it as it is